modules/db/src/test/java/org/mule/module/db/integration/function/DbCreateArrayFunctionTestCase.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.db.integration.function;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.mule.module.db.integration.model.Contact.CONTACT2;
+import static org.mule.module.db.integration.model.Region.NORTHWEST;
+import org.mule.api.MuleMessage;
+import org.mule.api.client.LocalMuleClient;
+import org.mule.module.db.integration.model.AbstractTestDatabase;
+
+import java.sql.Struct;
+
+import javax.sql.DataSource;
+
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DbCreateArrayFunctionTestCase extends AbstractDbFunctionTestCase
+{
+
+    public DbCreateArrayFunctionTestCase(String dataSourceConfigResource, AbstractTestDatabase testDatabase)
+    {
+        super(dataSourceConfigResource, testDatabase);
+    }
+
+    @Override
+    protected String[] getFlowConfigurationResources()
+    {
+        return new String[] {"integration/function/create-array-udt-config.xml"};
+    }
+
+    @Before
+    public void setupStoredProcedure() throws Exception
+    {
+        final DataSource dataSource = getDefaultDataSource();
+        testDatabase.createStoredProcedureGetZipCodes(dataSource);
+        testDatabase.createStoredProcedureGetContactDetails(dataSource);
+        testDatabase.createStoredProcedureUpdateZipCodes(dataSource);
+        testDatabase.createStoredProcedureUpdateContactDetails(dataSource);
+    }
+
+    @Test
+    public void createsDefaultTypeArray() throws Exception
+    {
+        LocalMuleClient client = muleContext.getClient();
+
+        MuleMessage response = client.send("vm://createsDefaultTypeArray", NORTHWEST.getZips(), null);
```

Comment:
No really required, I did it in this way as all the tests in the module use the same approach. I can change it